### ISSUE TITLE
Library update - bvanheu new files

### ISSUE THREAD

**Issue by mederux** [slaytonrnd/CartridgeWriter#10]
Your project is very good it help us all with the reading and re-writting of our eeproms...

Do you think it is possible to update the libraries?
they have another machines added as well some code updates.

Thank you very much.

**Comment by slaytonrnd** [comment]
Yes I will take a look and update; it will take some time for me to get to it.  Maybe by Friday.

**Comment by mederux** [comment]
Wow! Thank you for your fast reply!
That is great! 

Just to understand how exactly your software works, it reads the EEPROM and use the decrypt created by bvanheu and them write back the updated values? 
Or it reads, we have to run the decryption tool and them supply the new values on your program to be written on the EEPROM?

Also, I saw the wiring on the Arduino however on the Uprint the chip is different you can see on the images bellow that it has 4 contact areas (even if it only uses 2) do you think it is possible to use it anyway?

![new image1](https://cloud.githubusercontent.com/assets/12435400/7632800/1dd196ce-fa48-11e4-9d35-39d386106d2c.JPG)
![new image2](https://cloud.githubusercontent.com/assets/12435400/7632801/1dd3d24a-fa48-11e4-8f55-602a42a5d834.JPG)

Thank you very much again!

**Comment by ajquick** [comment]
Would you be able to compile a new release?
